roles/KubevirtTemplateValidator/defaults/main.yml

```diff
@@ -1,6 +1,3 @@
 ---
 # defaults file for KubevirtTemplateValidator
 wait: true
-validator_image: kubevirt-template-validator
-# this comes from the CR, we must diverge from validator_$SOMETHING standard naming
-version: "{{ validator_tag |default('v0.6.2') }}"
```

Comment:
Here I would just use version: latest, but I would keep it here.

Reply:
ack, see above